cycle_for/task_02.py

```diff
@@ -0,0 +1,12 @@
+# Даны два целых числа A и В. Выведите все числа от A до B включительно, в порядке возрастания, если A < B,
+# или в порядке убывания в противном случае.
+
+A = int(input())
+B = int(input())
+
+if A <= B:
```

Comment:
You can do it this way to make it shorter:
```
step = 1 if A < B else -1
for number in range(A, B + step, step):
    print(number)
```

Reply:
Okey